docs/da/index.md

```diff
@@ -0,0 +1,51 @@
+---
+description: Den officielle dokumentation for AtlasOS
+icon: material/home
+---
+
+# Velkommen til Atlas' Dokumentation
+
+<div class="grid cards" markdown>
+
+-   :material-power:{ .lg .middle } __Installer AtlasOS__
+
+    ---
+
+    start op med at få AtlasOS sat op på dit system på blot et par minutter.
```

Comment:
Is "start" not supposed to be with an upper-case S?

Reply:
Yeah I didn't notice that, I'll fix it on the next version